src/main/scala/com/lightbend/kafkalagexporter/MainApp.scala

```diff
@@ -32,10 +32,19 @@ object MainApp extends App {
     val clientCreator = (cluster: KafkaCluster) =>
       KafkaClient(cluster, appConfig.clientGroupId, appConfig.clientTimeout)(kafkaClientEc)
     val server = new HTTPServer(appConfig.port)
-    val endpointCreator = () => PrometheusEndpointSink(Metrics.definitions, appConfig.metricWhitelist,
-      appConfig.clustersGlobalLabels(), server, CollectorRegistry.defaultRegistry)
-
+    val endpointCreators = List(
+      KafkaClusterManager.NamedCreator(
+        "prometheus-lag-reporter", 
+        (() => PrometheusEndpointSink(
+          Metrics.definitions, appConfig.metricWhitelist, appConfig.clustersGlobalLabels(), server, CollectorRegistry.defaultRegistry
+        ))
+      ),
+      KafkaClusterManager.NamedCreator(
```

Comment:
I'd prefer to only create the `graphite-lag-reporter` if its config exists, otherwise it seems like a waste to create it at all and report all metrics to it.